docs/configuration/http_conn_man/route_config/traffic_splitting.rst

```diff
@@ -0,0 +1,192 @@
+.. _config_http_conn_man_route_table_traffic_splitting:
+
+Traffic Shifting/Splitting
+===========================================
+
+Envoy's router can split traffic to a route in a virtual host across
+two or more upstream clusters. There are two common use cases. The first use
```

Comment:
I would do

There are two common use cases:

1.  Version Upgrades: where traffic to a route is...
2.  A/B testing or multivariate testing: where `two or more versions`...